website/_enemies/mansion/gargoyle-ground.md

```diff
@@ -13,3 +13,4 @@ game_name: enemy_gargoyle_big
 sprite_folder_name: bgargoyle
 ---
 
+Due to the long startup of their attack, can commonly cause "Ghost punches" to occur, where the attack hitbox becomes active after they have already died.
```

Comment:
This is a general thing that happens in the game, such as if the player dies on spikes while performing an attack. Not sure if this should be covered in a tech section, either instead or in addition to?

Reply:
I agree, it should be covered in tech instead. To be honest, I just couldnt think of anything else remarkable, and didnt want to leave the field blank.

Reply:
Added to TODO on Discord